test/test_autograd.py

```diff
@@ -4457,6 +4457,35 @@ def test_checkpointing(self):
         mean_combined = torch.stack(feat_combined).mean()
         mean_combined.backward()
 
+#    @unittest.skipIf(not torch.cuda.is_available(), "Test requires CUDA")
```

Comment:
This can be removed?
Should we have a second test that tests the same thing on cuda?